tests/integration/telemetry/stackdriver/vm/testdata/server_request_count.json.tmpl

```diff
@@ -25,5 +25,12 @@
           "source_workload_name": "client-v1",
           "source_workload_namespace": "{{ .EchoNamespace }}"
        }
+    },
+    "resource": {
+       "labels": {
+          "container_name": "app",
+          "namespace_name": "{{ .EchoNamespace }}"
+       },
+       "type": "k8s_container"
```

Comment:
this resource type seems incorrect for a VM test. we need to address this in a follow-on PR.

Reply:
@douglas-reid : I think I need to do in this PR itself as test is still failing.. what is the right type for vm test?

Reply:
for vms on GCP, we want this ultimately to be `gce_instance`.